### PR TITLE
Fixture for Machine & MachineSet so the tests pass when none exists

### PR DESCRIPTION
**Pull request by miyamotoh**
Adding the necessary logic for `Machine` and `MachineSet` fixture injection and cleanup before  and after the Cypress' `other-routes` suite.

Signed-off-by: Hiro Miyamoto <miyamotoh@us.ibm.com>

### CODE REVIEW

File: frontend/packages/integration-tests-cypress/tests/crud/other-routes.spec.ts [hunk 1]
```diff
@@ -1,18 +1,57 @@
...
+      yamlEditor.clickSaveCreateButton();
+      cy.get(errorMessage).should('not.exist');
+    });
+  });
+};
+
+const deleteFixture = (kind: string) => {
+  cy.visit('/');
+  nav.sidenav.clickNavLink(['Compute', `${plural(kind)}`]);
+  listPage.filter.byName(testName);
```
[the remark this answers is not in the document]
Comment (dtaylor113) [reply frontend/packages/integration-tests-cypress/tests/crud/other-routes.spec.ts:33]: Hi @miyamotoh, I don't see this change yet ^^.   I don't believe `listPage.filter.byName(testName)` will pass CI tests as is.

**Review by dtaylor113 — CHANGES_REQUESTED** [review]

**Review by spadgett — CHANGES_REQUESTED** [review]
/hold

Let's mock the HTTP responses instead.

https://docs.cypress.io/guides/guides/network-requests

**Comment by spadgett** [comment]
I think it would be better to stay away from fixtures that can add nodes to the cluster or leave things in a bad state. I think mocking in this case is better, or even just skipping the tests if there are no machines.

**Comment by dtaylor113** [comment]
> or even just skipping the tests if there are no machines.

Hi @spadgett, is `Cypress.env('openshift') === true` be set on a Power cluster?  We use this env. var in our `resource-crud.spec.ts` to only test against OpenShift resource types.  I looked in our `test-prow-e2e.sh` script and I don't see us exporting a `openshift` env. var.?  Was wondering if we could use it to skip the Machine other routes url.

**Pull request closed without merging**